airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -29,6 +29,8 @@
 # the initial values should be saved once and tracked for each stream, inclusivelly.
 GoogleAnalyticsQuotaHandler: GoogleAnalyticsApiQuota = GoogleAnalyticsApiQuota()
 
+PAGE_SIZE = 100000
```

Comment:
@mpetrykin can you add a comment directing to where you can find this info?

Reply:
done :heavy_check_mark: